deploy/ecommerce-eks.yaml

```diff
@@ -26,7 +26,6 @@ ecommerce:
       needs:
         - instance
       args:
-        netcat_host_ip: !secret attacker.ip
```

Comment:
Need to remove this from required_secrets

Reply:
Great catch.. fixed!!